legacy/ui/legacy/src/main/res/values/strings.xml

```diff
@@ -343,6 +343,11 @@
     <string name="sent_folder_label">Sent folder</string>
     <string name="trash_folder_label">Trash folder</string>
     <string name="archive_folder_label">Archive folder</string>
+    <string name="archive_granularity_label">Archive options</string>
```

Comment:
I tried to match Thunderbird Desktop here but really there's only one option, so maybe this copy should be thought through once more. This is more stable if the area does get more features, and in that case "Archive options" would be fine, but currently it's a bit meh.